app/src/main/java/com/github/sdp/ratemyepfl/activity/course/CourseReviewListActivity.kt

```diff
@@ -81,26 +68,4 @@ class CourseReviewListActivity : AppCompatActivity() {
         resultLauncher.launch(intent)
     }
 
-    private fun createOnScrollListener(

```

Comment:
I found the button my bad. I was wondering why you removed the scroll feature.

Reply:
I refactored this view to a recycler view, so a scroll listener is not needed anymore. The scroll should work without it. I will test it again just to be sure

Reply:
Ahhh, actually it was only for the button to shrink or extend. But if you want to remove this feature, feel free to do it!

Reply:
Ohh I will put it back then :), it was a cool feature.